core/src/main/java/org/jdbi/v3/core/result/EmptyResultSetMetaData.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.result;
+
+import java.sql.ResultSetMetaData;
+
+public class EmptyResultSetMetaData implements ResultSetMetaData {
```

Comment:
Could this class be made package-private, or moved to `internal`?  I don't see it having much value as public API.